scripts/run.py

```diff
@@ -128,73 +132,98 @@ def plot_metrics(directory_name):
     "-u", "--url", "url", help="Online data link.", default=None, required=False
 )
 @click.option("-m", "--model-point", "model_point", help="Model point.", required=False)
-def main(backend, path, url, model_point):
+@click.option("-n", "--number", "number", help="Number.", default="0", required=False)
+@click.option("-mm", "--mode", "mode", help="Mode.", default="fast", required=False)
+def main(computation, backend, path, url, model_point, number, mode):
     """
     Automatic process of taking pyhf computation.
 
     Usage:
 
-      $ python run.py [-b] [-p] [-u] -m
-
-    Examples:
+    $ python run.py -c [-b] [-p] [-u] [-m] [-n] [-mm]
 
-      $ python run.py -b numpy -u https://www.hepdata.net/record/resource/1267798?view=true -m [750,100]
-      $ python run.py -u https://www.hepdata.net/record/resource/1267798?view=true -m [750,100]
-      $ python run.py -b numpy -p 1Lbb-likelihoods-hepdata -m [750,100]
+  Examples:
 
-    More information:
+    $ python run.py -c mle -b numpy -u https://www.hepdata.net/record/resource/1267798?view=true -m [750,100]
+    $ python run.py -c mle -u https://www.hepdata.net/record/resource/1267798?view=true -m [750,100]
+    $ python run.py -c mle -b numpy -p 1Lbb-likelihoods-hepdata -m [750,100]
+    $ python run.py -c interpolation -b jax -n 0 -mm fast
+    $ python run.py -c interpolation -b numpy -n 0 -mm slow
```

Comment:
This is looking good, but @matthewfeickert can also offer some examples once the packaging PR is in on how to split out the CLI more. :+1: